organization/_posts/2020/2020-09-30-packaging.md

```diff
@@ -0,0 +1,115 @@
+
+Apologies: 
+
+## Introduction (Ben)
+- As always, ideas or requests for presentations/topical meetings welcome! Just contact Ben, Serhan and Martin.
+    - Likely have updates/tutorials on some packaging tools in next month or two (will discuss offline/through mailing list)
+- Please note upcoming [HSF/WLCG Virtual Workshop, 19-24 November](https://indico.cern.ch/event/941278/)
+    - Especially [call for abstracts](https://indico.cern.ch/event/941278/abstracts/) on any software/development R&D
+
+
+## An Introduction to Modern CMake (Henry)
+- One main take-home: links on the front page to blog and training links
+- Talk is interactive, so can follow along, or revisit using the link on the agenda
+- Based on CMake 3.18
+- Example on Rake
+    - Knows what to run, but less ...
+- CMake = Build system generator
+    - But can use it to trigger build/install
+- Core concepts
+    - minimal script
+    - separation of build/source dirs
+- Wide range of buildsystems for C/C++
+    - cmake, scons, meson, bazel 
+    - Other languages have their own (e.g. Rust)
+- CMake has become a standard tool
+    - Wide IDE support (either in CMake, or via IDEs themselves)
+    - Libraries provide support for integration
+- Custom buildsystems going away
+    - e.g Qt6, Boost
+    - Google dual support Bazel, CMake
+- Growing support/improvements in things like Thrust/TBB/PyBind11
+    - Attila: Isn't TBB still using plain Make? Henry: main thing is that TBB installs a "TBBConfig.cmake" file that allows easy use of TBB in CMake projects.
+- "Modern" CMake roughly from 2014 with CMake 3.0
+    - "Target" based
+    - CMake 3.12 "more modern cmake", unified target behaviour
+- CMake backward compatibility with policies
+    - The `cmake_minimum_required` command
+    - CMake 3.12 introduced version ranges for this
+    - Do/Don't recommendation
+    - Chris: Also minimum version determined by needs for commands (e.g. subcommands in list)
+    - Henry: yes, key thing is the "Do" on testing with lowest version you want to support
+    - Example of a bad policy in CMake 3.9:
+        - breaks IPO
+    - Thus the recommendation on don't setting the min version low without a range
+    - Attila: big problem is updating min version through a project's file
+    - Henry: one recommendation here (though CMake didn't do this themselves) is not to set minimum required on anything other than `CMakeLists.txt`.
+- Minimum versions by OS (e.g. "out-the-box" on CentOS etc), vs CMake features (what it can do)
+- Installing CMake now very easy across all platforms (inc. GitLab/Hub CI)
+    - See the installing page on the website
+- Modern cmake has better CLI, so can run underlying configure/build/test/install directly through the `cmake` command
```

Comment:
I don't know if you care about consistently using the CMake capitalisation, but here you said "cmake"